src/components/CertificateTemplates/tlds/sg/edu/smu/degree-cert-template/certificate.js

```diff
@@ -0,0 +1,181 @@
+import PropTypes from "prop-types";
+import { get } from "lodash";
+
+const Template = ({ certificate }) => {
+  // Declaring what variables will be available to the template from the certificate
+  const certificateName = get(certificate, "name");
+  const honors = get(certificate, "additionalData.honors", "\u00a0");
+  const recipientName = get(certificate, "recipient.name");
+  const awardDate = get(certificate, "additionalData.awardDate");
```

Comment:
think there's a schema field for this

Reply:
The schema field was a date data type, and we have a data source field that is a pre-formatted string with the required date format. Can we keep this custom field?

Reply:
you can use the date data type and parse it into your required format subsequently in the template code. the reason we use a standard date type is so that it is machine readable